src/ai/backend/manager/api/gql/agent/types.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+from strawberry.relay import Node, NodeID
+from strawberry.scalars import JSON
+
+from ai.backend.manager.data.agent.types import AgentStatus
+from ai.backend.manager.models.rbac.permission_defs import AgentPermission
+
+
+@strawberry.type(description="Resource-related information for an agent")
+class AgentResourceInfo:
+    available_slots: JSON
+    occupied_slots: JSON
+    compute_plugins: JSON
```

Comment:
As we discussed before, we decided to unify the terminology as `capacity = free + used`, so please unify it while creating the new API.

Reply:
Please check the review.